code/modules/mapping/random_rooms.dm

```diff
@@ -1479,3 +1479,139 @@
 	centerspawner = FALSE
 	template_height = 11
 	template_width = 5
+
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
+	mappath = "_maps/RandomRooms/10x10/skwh_rdm158_abandonedviro.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 3
+
+/datum/map_template/random_room/skwh_rdm157
+	name = "Slime Storage"
+	room_id = "skwh_rdm157_slimestorage"
```

Comment:
Besides the overabundance of virus samples, this isn't terribly imbalanced.

Reply:
aight i'll replace all the blood in this room with fresh blood as i have many more maps to focus on in putting in more dresh blood then dried blood, plus i wanna help fix this PR real quick before i get back into working on some other stuff i want to work on so yeah